src/main/resources/org/jenkinsci/plugins/xunit/pipeline/XUnitResultStep/config.jelly

```diff
@@ -0,0 +1,28 @@
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2014, Gregory Boissinot, Nikolas Falco
```

Comment:
This file is new fix license with your name

Reply:
Whoops, sorry.